pyhomematic/devicetypes/sensors.py

```diff
@@ -190,6 +190,27 @@ class MotionV2(Motion, HelperSabotage):
     """Motion detection version 2."""
     pass
 
+class MotionIP(HMBinarySensor, HMSensor, HelperLowBat, HelperSabotage):
+    """Motion detection indoor (rf ip)"""
+
+    def __init__(self, device_description, proxy, resolveparamsets=False):
+        super().__init__(device_description, proxy, resolveparamsets)
+
+        # init metadata
+        self.BINARYNODE.update({"MOTION_DETECTION_ACTIVE": [1]})
+        self.BINARYNODE.update({"MOTION": [1]})
+        self.SENSORNODE.update({"ILLUMINATION": [1]})
+
+    def is_motion(self, channel=None):
+        """ Return True if motion is detected """
+        return bool(self.getBinaryData("MOTION", channel))
+
+    def is_motion_detection_active(self, channel=None):
+        return bool(self.getBinaryData("MOTION_DETECTION_ACTIVE", channel))
+
+    def get_brightness(self, channel=None):
+        """ Return brightness from 0 (dark ) to 255 (bright) """
```

Comment:
I'm not sure about this one, is there any way to detect the range over XML RPC?
In "practice" I've got only values between 0 and 18